components/brave_extension/extension/brave_extension/manifest.json

```diff
@@ -64,6 +68,16 @@
       ],
       "run_at": "document_start",
       "all_frames": true
+    },
+    {
+      "matches": [
+        "*://*/*"
+      ],
+      "run_at": "document_start",
+      "match_about_blank": true,
```

Comment:
this doesn't seem right, why would we care about about:blank pages?

Reply:
this is imported from their extension cc @remusao

Reply:
I will double-check but I think it might not be required. I will update here once I can confirm.

Reply:
I think it should be safe to remove the `match_about_blank` option. It does not seem to be required by human-web.

Reply:
fixed